### ISSUE TITLE
2.4 changelog 

### ISSUE THREAD

**Issue by Cellule** [enquirer/enquirer#434]
Can you add a proper changelog entry for the recent 2.4 release.
Real quick it looks like a bunch of PRs have been merged and then a release was made.

I can somewhat look over the commit list to figure out the change, but this is highly suspicious from a project that had no new versions in over 3 years
I was considering this project pretty much unmaintained and was considering moving to another project.
Is it going to be actively maintained now ?

**Comment by jonschlinkert** [comment]
> Real quick it looks like a bunch of PRs have been merged and then a release was made.

Yes, a bunch of PRs were merged. The vast majority were typos in the docs and typescript typings with a couple of bug fixes. No breaking changes. 

Do you have any specific questions about specific parts of the code that you're having issues with?

**Comment by Cellule** [comment]
I don't have any particular issue, but I'd expect the CHANGELOG.md file to reflect the changes included in the new version to make it easier to evaluate the nature and potential risk of updating (if any)

**Comment by tbeseda** [comment]
I'm stoked to see updates to `enquirer`. It would be nice to see that there weren't breaking changes reflected in the existing CHANGELOG.md -- just for a bit of confidence in even testing an update. my two cents.
Thanks for the work on enquirer.
